constraints.md

```diff
@@ -5,140 +5,125 @@ category: reference
 aliases: ['/docs/dev/constraints/','/docs/dev/reference/sql/constraints/']
 ---
 
-# Constraints
+# Constraint
```

Comment:
I think this should be plural, since there is more than one type of constraint.

Reply:
Updated.